### PR TITLE
added dalet ℸ smbol to dictionary

### LINKED ISSUE #44: Add daleth
The following symbol can be added to the dictionary.

Please read the formatting guidelines explained in #5 to contribute!

`\daleth	ℸ	`

### PR DESCRIPTION
**Pull request by prahuljose**
resolves #44

### CODE REVIEW

**Comment by DenverCoder1** [comment]
Could you make sure the tab format matches the correct format (see issue #5)? You can copy the tab symbols from other lines.

You can update this PR by adding commits to your branch.

**Comment by DenverCoder1** [comment]
Also, if you can, please change the name from dalet to `daleth` to match the LaTeX shortcut as seen here:

https://www.tutorialspoint.com/tex_commands/daleth.htm

**Comment by prahuljose** [comment]
> Also, if you can, please change the name from dalet to `daleth` to match the LaTeX shortcut as seen here:
> 
> https://www.tutorialspoint.com/tex_commands/daleth.htm

hey, updated from `dalet` to `daleth`

File: dictionary.txt [hunk 1]
```diff
@@ -926,3 +926,5 @@ _rp	₎
 \doublebarvee	⩢	
 \doublebarwedge	⩞	
 \perspcorrespond	⩞	
+\daleth  ℸ 
+
```
Comment (DenverCoder1) [inline dictionary.txt:930]: Make sure to correct the spaces to tabs also
```suggestion
\daleth	ℸ	
```

Comment (prahuljose) [reply dictionary.txt]: done!

**Review by DenverCoder1 — APPROVED** [review]
Looks good! Thanks for contributing! :tada:

**Pull request merged**